azkaban-exec-server/src/main/java/azkaban/execapp/action/KillExecutionAction.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.execapp.action;
+
+import azkaban.ServiceProvider;
+import azkaban.execapp.FlowRunnerManager;
+import azkaban.trigger.builtin.SlaChecker;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+
+import azkaban.trigger.TriggerAction;
+
+public class KillExecutionAction implements TriggerAction {
+
+  public static final String type = "KillExecutionAction";
```

Comment:
i'm following legacy convention here for being consistent. I think action implementation needs to be refactor and type should be enum. but it will requires changes on multiple classes and would be too big to fit into this PR.